ldblink.cpp

```diff
@@ -1,16 +1,3 @@
-// (c) Copyright [2022-2023] Micro Focus or one of its affiliates.
-// Licensed under the Apache License, Version 2.0 (the "License");
-// You may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
Why Copyright is removed?

Reply:
Yes I explained the basics. He is not used to it so we are going to do it step by step. This PR is from a branch in mfelici/dblink into vertica:main which is the correct thing to do. I asked him to add many reviewers so that he can get a lot of feedback and then, I will help him to address things related to git.

Reply:
yes, we did it together step by step. Created a new branch, add reviewrs, etc. Next time will be better and we will pay more attention to all these GH formalities. Now, let's make it easy... 
STEP 1: one or more reviewrs accept my pull requests as is
STEP 2: someone will re-apply this copyright notice. 
OK? Are there any comment on the "real code"?

Reply:
@mfelici, can you please just keep the copyright and push the code again?

Reply:
@NerdLogic can you please review the code of Maurizio to see if he did some mistakes? On my side, I'll check it now.

Reply:
> @mfelici, can you please just keep the copyright and push the code again?

Ok. Will do this way when I willl come back from holidays (Jan 9th)... 
**First** I will revoke this pull request. 
**Then** on my side (meaning mfelici/dblink) I am going to add the copyright notice to the code and (?) remove the changelog (not sure about this: maybe we can keep the easy-to-read Changelog AND use Releases at the same time)
And, **finally**, create a new pull request. Ok?

Reply:
@mfelici You don't need to revoke this pull request, if you push further commits to the branch [session_param](https://github.com/mfelici/dblink/tree/session_param), then changes will reflect on this pull request.

Reply:
@mfelici when you come back from holidays we could have another call if you have any doubts. Just let me know if you need some help.

Reply:
> @mfelici You don't need to revoke this pull request, if you push further commits to the branch [session_param](https://github.com/mfelici/dblink/tree/session_param), then changes will reflect on this pull request.

ok, done. Added the license info at the beginning of the library (just below the VIM modeline as it should be) and committed the changes in session_param branch of mfelici/dblink.

Reply:
Thanks @mfelici. I think you were  also asked to delete CHANGELOG.md.

Reply:
Yes, I remember I have been asked to remove the CHANGELOG but I find useful to have all changes in one place. Why should we get rid of it? Can you please help me understanding why should we remove it?

Reply:
Because there are better ways to track changes and ultimately each release will come with its set of changes on its description.

Reply:
We can have releases and you are free to implement these "better ways" but I like the idea to have ALL changes listed in ONE place without having to browse here and there. Maybe one day or another I will appreciate these "better ways". In the meantime... let's use what we have: the CHANGELOG. Ok?

Reply:
Yes I definitely agree. The tool I propose to use generate a CHANGELOG. But it automatically updates it for you. Give me a change to show it to you. You will like it. I too like to have a CHANGELOG

Reply:
Take a look at this CHANGELOG for example https://github.com/vertica/vertica-kubernetes/blob/main/CHANGELOG.md. That tool creates it for you and maintain it. You even have links to Pull Requests.